index.js

```diff
@@ -212,7 +212,7 @@ const decorateServerConfig = (config, options) => {
   });
 }
 
-const postcssPlugins = ({ cssSelectorPrefix }, autoprefixerConfig) => 
```

Comment:
Strange, I would have thought the linter would have picked this up, since we're shadowing an existing variable 🤔

Reply:
Might be worth double checking the lint rules.